packages/mdc-chips/chip-set/foundation.js

```diff
@@ -76,12 +76,10 @@ class MDCChipSetFoundation extends MDCFoundation {
   }
 
   /**
-   * Handles a chip interaction event
-   * @param {!Object} evt
-   * @private
+   * Manages the selection state of the chip set, given a chip that was just selected/deselected.
+   * @param {!MDCChip} chip
    */
-  handleChipInteraction_(evt) {
-    const {chip} = evt.detail;
+  manageSelection(chip) {
```

Comment:
Meant to respond here earlier...
This looks great! This will also make it easier to collapse chip and chip-set into one controller, if we ever decide to do that.